recipes/graph/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "graph" %}
+{% set version = "1.11" %}
+{% set sha256 = "00432f3007b4cb2a8d6f2e021fd5625a79e307d6532971719ca6ab81dd0ae752" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/bjoern-andres/graph/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+  patches:
+    - hdf5_fixes.diff
+    - install_fixes.diff
+    - msvc_fixes.diff
+
+build:
+  number: 0
+  skip: true        # [win32 or (win and py2k)]
```

Comment:
This would create two builds for `py35` and `py36`. Since they use the same MSVC version skip one